packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -21,13 +143,95 @@ export default {
 
 .c-mediaElement {
     display: flex;
+    width: 100%;
+}
+.c-mediaElement--stack {
+    flex-direction: column;
+}
+.c-mediaElement--stack.c-mediaElement--reverse {
+    flex-direction: column-reverse;
+}
+
+.c-mediaElement-content {
+    display: flex;
+    flex-direction: column;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    flex: 1 1 0;
+
+    &--center {
+        align-items: center;
+        text-align: center;
+    }
+    &--left {
+        align-items: flex-start;
+    }
+    &--right {
+        align-items: flex-end;
+    }
+}
+
+.c-mediaElement-title.c-mediaElement-fontSize {
+    &--sm {
+        @include font-size(heading-s);
+    }
+    &--md {
+        @include font-size(heading-m);
+    }
+    &--lg {
+        @include font-size(heading-l);
+        margin-bottom: spacing(x0.5);
+    }
+    &--xl {
+        @include font-size(heading-xl);
+        margin-bottom: spacing(x0.5);
+    }
+    &--xxl {
+        @include font-size(heading-xxl);
+        margin-bottom: spacing(x0.5);
+    }
+}
+
+.c-mediaElement--stack .c-mediaElement-title {
+    margin-top: spacing(x3);
+}
+
+.c-mediaElement-text.c-mediaElement-fontSize {
+    @include font-size(body-l);
+    margin-top: spacing(x0.5);
+    &--sm {
+        @include font-size(body-s);
+    }
+    &--md {
+        @include font-size(body-l);
+    }
+    &--lg {
+        @include font-size(subheading-s);
+        margin-bottom: spacing(x0.5);
+    }
+    &--xl {
+        @include font-size(subheading-s);
+        margin-bottom: spacing(x0.5);
+    }
+    &--xxl {
+        @include font-size(subheading-l);
+        margin-bottom: spacing(x0.5);
+    }
+}
+
+.c-mediaElement-imgWrapper {
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 0;
+
+    &--center {
+        align-items: center;
+    }
+    &--left {
+        align-items: flex-start;
+    }
+    &--right {
+        align-items: flex-end;
+    }
```

Comment:
Same as above. Would adjust this to be longhand for better codebae searchability.
```suggestion
.c-mediaElement-imgWrapper {
    display: flex;
    flex-direction: column;
    flex: 1 1 0;
}

    .c-mediaElement-imgWrapper--center {
        align-items: center;
    }
    .c-mediaElement-imgWrapper--left {
        align-items: flex-start;
    }
    .c-mediaElement-imgWrapper--right {
        align-items: flex-end;
    }
```

Reply:
I have modified these for alignment to be as described, but I am now handling the font sizing differently